internal/cli/command/cluster/feature/features/dns/activate.go

```diff
@@ -446,12 +489,12 @@ func getFeatureSpecDefaults(banzaiCLI cli.Cli, provider Provider) (DNSFeatureSpe
 				Provider: &Provider{
 					Name: dnsBanzaiCloud,
 				},
+				// defaults to the clusterdomain
+				DomainFilters: strings.Split(clusterDomain, ","),
```

Comment:
`clusterDomain` should not contain `,`s, so why split it by them.

Reply:
this is a comma separated list

Reply:
are you sure? [this line](https://github.com/banzaicloud/banzai-cli/pull/154/files#diff-7de3a3940cac6077e806dfe69d29621eR489) tells otherwise

Reply:
thx